modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,160 @@
+With Server 10.2 a new permission for public links on folders is available for users. 
+It lets recipients view, download, and upload documents, yet prevents them from make changes (e.g., rename, move, collaborative editing). 
+The new permission, named `Download / View / Edit`, has been added as a replacement for the former `Download / View / Upload`.
+It has been changed and can now be regarded as a public file drop to distribute and gather information with a single link, while preventing recipients from changing contents.
+
+=== Storage Encryption with Master Key in HSM
+
+With version 10.2, ownCloud Server officially supports storage encryption with master keys stored in hardware security modules (HSM). 
+In contrast to regular master key-based storage encryption, which stores the keys on the storage, storage encryption with keys in an HSM allows administrators to completely prevent anyone with access to the storage from accessing the data stored in ownCloud. 
+
+As a result, the bundled `encryption` app has been updated to support HSM, and a standalone service (`hsmdaemon`) that connects ownCloud Server and the HSM device is now available within ownCloud Enterprise Edition. 
+To get started with storage encryption and HSM, https://owncloud.com/contact/[please get in touch with us].
+For more information around the different encryption types ownCloud offers, consider https://oc.owncloud.com/rs/038-KRL-592/images/Whitepaper_Data_Protection_and_Data_Secrecy_in_ownCloud_EN.pdf[this whitepaper].
+
+=== Background Job for Change Detection of Nested Federated Shares
+
+When using federation to share data across ownCloud instances, deeply nested folders (e.g., folders with many sub-items) https://github.com/owncloud/docs/issues/856[are not discovered automatically for performance reasons]. 
+This leads to several issues such as the ownCloud Desktop Client not being able to synchronize newly added or changed content unless the user navigates down the hierarchy using the web interface, which manually triggers content discovery. 
+
+Also, the size of such folders can't be calculated, showing "Pending" instead, until the discovery is manually triggered. 
+To help alleviate this problem, a new `occ` command has been introduced.
+It can be executed regularly as a background job to discover federated shares (`occ incoming-shares:poll`). 
+This is aimed at handling this issue while providing the means for administrators to control resource usage.
+
+When using federation, it is recommended to execute `occ incoming-shares:poll` regularly xref:configuration/server/background_jobs_configuration.adoc#cron-jobs[using Cron jobs]. 
+The time interval to choose between executions is a trade-off between the availability of changes in federated shares and resource consumption, which naturally depends a lot on the number of federated shares and the frequency of changes within those shares. 
+
+As a middle ground, we recommend executing the command every 15 minutes. 
```

Comment:
We still need to verify this recommendation value.

Reply:
I'm updating this, based on https://github.com/owncloud/docs/issues/856#issuecomment-481748713, from @VicDeo.